en/development/routing.rst

```diff
@@ -90,6 +90,8 @@ parameters::
         ['controller' => 'Users', 'action' => 'login'],
         ['_name' => 'login']
     );
+    // Be aware if you change your actual login route you must also update your
```

Comment:
This isn't really related to the example. It doesn't belong in a comment either.